v2/azure-arm.yaml

```diff
@@ -814,22 +826,35 @@ status:
 #
 # Available 'type' modifiers:
 #
-#   $renamedTo: <string>  // Gives the name this type will have in the next 
-#                         // version of the resource
+# $export: <bool>
+#     Requests that support for this resource type be generated.
+#     Automatically includes all other types required for this resource
+#     If not specified, false is assumed.
```

Comment:
minor: "assumed" here feels more wishy-washy to me than you possibly intend?

maybe: "If not specified, the default is false."

Reply:
Fixed.